### PR TITLE
chore: add example for catch bug on iOS

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

### CODE REVIEW

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/dialog/dialog.template.html [hunk 1]
```diff
@@ -90,6 +90,15 @@
         >
             <tui-dialog-example-5></tui-dialog-example-5>
         </tui-doc-example>
+
+        <tui-doc-example
+            id="mobile"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/dialog/dialog.template.html:95]: this page already has element with `id="mobile"`.
Let's take another id.

File: projects/demo/src/modules/components/dialog/dialog.component.ts [hunk 2]
```diff
@@ -89,6 +90,11 @@ export class ExampleTuiDialogComponent {
         HTML: example6Html,
     };
 
+    readonly example7: TuiDocExample = {
+        TypeScript: import('!!raw-loader!./examples/7/index.ts'),
+        HTML: import('!!raw-loader!./examples/7/index.html'),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/dialog/dialog.component.ts:95]: add `SearchDialogExampleComponent`-files (otherwise stackblitz will not work)

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/dialog/dialog.component.ts (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/dialog/dialog.component.ts:95]: fixed

File: projects/demo/src/modules/components/dialog/dialog.template.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/dialog/dialog.template.html:95]: fixed

**Pull request merged**